vendor/brave-ios/Ledger/BATBraveLedger.mm

```diff
@@ -1277,6 +1277,10 @@ - (void)setBooleanState:(const std::string&)name value:(bool)value
 - (bool)getBooleanState:(const std::string&)name
 {
   const auto key = [NSString stringWithUTF8String:name.c_str()];
+  if (![self.prefs objectForKey:key]) {
```

Comment:
Default to `NO` for unknown keys to support flags on desktop which are not currently supported on iOS